modules/kafka/kafka-dest-driver.h

```diff
@@ -59,9 +58,12 @@ void kafka_dd_set_message_ref(LogDriver *d, LogTemplate *message);
 void kafka_dd_set_flush_timeout_on_shutdown(LogDriver *d, gint shutdown_timeout);
 void kafka_dd_set_flush_timeout_on_reload(LogDriver *d, gint reload_timeout);
 void kafka_dd_set_poll_timeout(LogDriver *d, gint poll_timeout);
+
+gboolean kafka_dd_validate_topic_name(KafkaDestDriver *self, const gchar *name);
 gboolean kafka_dd_is_topic_name_a_template(KafkaDestDriver *self);
 rd_kafka_topic_t *kafka_dd_query_insert_topic(KafkaDestDriver *self, const gchar *name);
 LogTemplateOptions *kafka_dd_get_template_options(LogDriver *d);
+gboolean kafka_dd_init(LogPipe *s);
```

Comment:
This should be in the internal header IMO. If you move it to the internal header, rename it to `kafka-internal.h`